virttest/qemu_vm.py

```diff
@@ -225,6 +225,37 @@ def __init__(self, name, params, root_dir, address_cache, state=None):
         self.last_boot_index = 0
         self.last_driver_index = 0
 
+    def _get_cmdline_format_cfg(self):
+        """
+        get data from file or input from parameter and then convert data to dict
+        :return: style data in dict
+        """
+        def _file(filepath):
+            if not filepath:
+                raise ValueError("The filepath is empty!")
+            if not os.path.isabs(filepath):
+                vt_type = self.params.get("vt_type", "qemu")
+                filepath = data_dir.get_backend_cfg_path(vt_type, filepath)
+            with open(filepath, "r") as f:
+                content = f.read()
+            return content
+
+        def _string(content):
+            return content
+
+        def _default(dummy):
+            return "{}"
+
+        handler, value = self.params.get("qemu_cmdline_format_cfg", ":").split(
```

Comment:
> Please add the docstring for this parameter `qemu_cmdline_format_cfg` in the base.cfg https://github.com/avocado-framework/avocado-vt/blob/master/virttest/shared/cfg/base.cfg

Hello @YongxueHong , the `qemu_cmdline_format_cfg` was added at Kar project instead of avocado-vt project.  
Thanks!

Reply:
Hi @nickzhq 
yeah, I saw that, but we are adding this function in the avocado-vt project which is also an upstream project, so as to should give the docstring about it for the upstream.